jsonnet/components/console-plugin.libsonnet

```diff
@@ -0,0 +1,210 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
+
+function(params)
```

Comment:
@raptorsun A little bird to me that you are one of the jsonnet experts. It would be great if you   could please help review this jsonnet magic :)

Reply:
I didn't realise ksonnet was deprecated 🤦 .. I will convert everything to not use it.